Makefile

```diff
@@ -8,7 +8,7 @@ DATE ?= $(shell date -u --iso-8601=seconds)
 COMMIT ?= $(shell git log -1 --pretty=format:"%h")
 
 run-api-server: 
-	APISERVER_PORT=8080 go run cmd/api-server/main.go
+	APISERVER_PORT=8087 go run cmd/api-server/main.go
```

Comment:
This port is used by kubectl kubtest direct client - and  need to be changed too